price-feeder/oracle/provider/okx.go

```diff
@@ -162,47 +178,73 @@ func (p *OkxProvider) setTickerPair(tickerPair OkxTickerPair) {
 
 // subscribeTickers subscribe to all currency pairs
 func (p *OkxProvider) subscribeTickers(cps ...types.CurrencyPair) error {
-	topics := make([]SubscriptionTopic, len(cps))
+	topics := make([]OkxSubscriptionTopic, len(cps))
 
 	for i, cp := range cps {
 		instId := getInstrumentId(cp)
-		topics[i] = newSubscriptionTopic(instId)
+		topics[i] = newOkxSubscriptionTopic(instId)
 	}
 
-	subsMsg := newSubscriptionMsg(topics...)
+	subsMsg := newOkxSubscriptionMsg(topics...)
 	return p.wsClient.WriteJSON(subsMsg)
 }
 
-func (tickerPair OkxTickerPair) toTickerPrice() (TickerPrice, error) {
-	price, err := sdk.NewDecFromStr(tickerPair.Last)
-	if err != nil {
-		return TickerPrice{}, fmt.Errorf("failed to parse Okx price (%s) for %s", tickerPair.Last, tickerPair.InstId)
-	}
+func (p *OkxProvider) resetReconnectTimer() {
+	p.reconnectTimer.Reset(okxPingCheck)
+}
 
-	volume, err := sdk.NewDecFromStr(tickerPair.Vol24h)
+// reconnect closes the last WS connection and create a new one
```

Comment:
awesome docs!! just one nit: 
```suggestion
// reconnect closes the last WS connection and creates a new one.
```